README.md

```diff
@@ -1,4 +1,65 @@
-# Overview
+# Entando Hub Bundle
+
+## Overview
 This is the parent repository for the Entando Hub project which currently contains two deployable sub-projects
 * `application` which has the MFEs and microservices for the Entando Hub application
 * `content` which has the non-compiled components such as pages, page templates, and fragments.
+
+## Getting Started
+
+### Deploy on Entando App
+
+Move inside one of the two folders `application` and `content`, which contain the bundles.<br/>
+You can build,deploy and install the bundles on Entando with commands of ent.
+
+```
+ent bundle pack
+ent bundle publish
+ent bundle deploy
+ent bundle install
+```
+
+For more details check the readme of `application` and `content`.
+
+## Configure Hub Registry in AppBuilder
+This sections is useful to connect an existing Entando Hub.
+From AppBuilder menu, you select `Hub`. At the top right, select `Select Registry` and `New Registry`.
+
+You can choose Name and configure the url endpoint.
+
+* example endpoint value for localhost:
+
+```
+http://localhost:8081/appbuilder/api
+```
+
+* example endpoint value for production: 
+
+```
+http://{{YOUR-HOSTNAME}}/entando-hub-application-{{BUNDLE-CODE}}/entando-hub-catalog-ms/appbuilder/api
+```
+
+You can use also Entando Hub from `entando.com`:
+```
+https://entando.com/entando-hub-api/appbuilder/api
+```
```

Comment:
I think we could delete this since it's going to adhere to the previous one when we will update it to bundle v5